notebooks/bootcamp-test-assignment.ipynb

```diff
@@ -0,0 +1,280 @@
+     "schema_version": 3,
+     "solution": true,
+     "task": false
+    },
+    "tags": [
+     "hide"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Import necessary Python packages\n",
+    "\n",
+    "# BEGIN SOLUTION\n",
+    "import os\n",
+    "import matplotlib.pyplot as plt\n",
+    "import pandas as pd\n",
+    "import earthpy as et\n",
+    "\n",
+    "# END SOLUTION\n",
+    "\n",
+    "### DO NOT REMOVE LINES BELOW ###\n",
+    "import matplotcheck.notebook as nb\n",
+    "import matplotcheck.timeseries as ts"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nbgrader": {
+     "grade": false,
+     "grade_id": "cell-6aebba45d37bfee4",
+     "locked": true,
+     "schema_version": 3,
+     "solution": false,
+     "task": false
+    }
+   },
+   "source": [
+    "### Question 2: Download Data and Plot\n",
+    "\n",
+    "1. Use `earthpy.data.get_data()` to download the following .csv file:\n",
+    "    * `boulder-precip-2007-to-2017-months-seasons.csv` from `url = https://ndownloader.figshare.com/files/12767933`\n",
+    "        * The data will download to `~/earth-analytics/data/earthpy-downloads/`.\n",
+    "        * This dataset contains a row for each month (starting with January through December) and contains a column for each year (2007 to 2017). \n",
+    "        * More information on `earthpy.data.get_data()` available at <a href=\"https://earthpy.readthedocs.io/en/latest/api/earthpy.io.html#earthpy.io.Data.get_data\" target=\"_blank\">this link</a>.        \n",
+    "2. Import the data to a `pandas dataframe` and create a line plot for **2013** using `matplotlib`. \n",
+    "    * Be sure to include:\n",
+    "        1. An appropriate plot title.\n",
+    "        2. Clear labels for the axes (including abbreviation for units of measurements where apppropriate)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "nbgrader": {
+     "grade": false,
+     "grade_id": "plot-2013",
+     "locked": false,
+     "schema_version": 3,
+     "solution": true,
+     "task": false
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Create plot for 2013\n",
+    "\n",
+    "# BEGIN SOLUTION\n",
+    "\n",
+    "# Download data \n",
+    "url = \"https://ndownloader.figshare.com/files/12767933\"\n",
+    "et.data.get_data(url=url)\n",
+    "\n",
+    "# Set working directory\n",
+    "os.chdir(os.path.join(et.io.HOME, 'earth-analytics'))\n",
+    "\n",
+    "# Import to pandas dataframe\n",
+    "precip_2007_to_2017_months = pd.read_csv(\"data/earthpy-downloads/boulder-precip-2007-to-2017-months-seasons.csv\")\n",
+    "\n",
+    "fig, ax = plt.subplots()\n",
+    "\n",
+    "# Add the x-axis and the y-axis to the plot\n",
+    "ax.plot(precip_2007_to_2017_months.months, precip_2007_to_2017_months.y2013, color=\"blue\")\n",
+    "\n",
+    "# Set plot title\n",
+    "ax.set(title=\"Monthly Precipitation in 2013 for Boulder, CO\")\n",
```

Comment:
@jlpalomino just a note that you can set the title x and y labels in one line here rather than calling `ax.set()` twice.  super minor

Reply:
Thanks for the reminder @lwasser!  I will update the notebook in my fork for future reference.